packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -547,8 +540,9 @@ export default {
                     logMethod: this.$logger.logInfo
                 });
             } catch (e) {
-                this.toggleDialogError();
-                throw new PlaceOrderError(e.message);
+                const { errorCode } = e.response.data;
+
+                throw new PlaceOrderError(e.message, errorCode === 'DuplicateOrder');
```

Comment:
`errorCode === 'DuplicateOrder'` is not very extensible in my opinion.
You already have this information in `checkout-issues.js`, so it would be better to drive this decision from that file, which should be our single source of truth in terms of handling problems (issues or errors).
```
  const { shouldShowInDialog } = checkoutIssues[errorCode];

  throw new PlaceOrderError(e.message, shouldShowInDialog);
```